src/unpackertool/plugins/upx/decompressors/decompressor_upxshit.h

```diff
@@ -8,6 +8,8 @@
 
 #include "unpackertool/plugins/upx/decompressors/decompressor_scrambler.h"
 
+using namespace retdec::utils;
```

Comment:
Same as the comment I made earlier. Never put `using namespace` in header files.